components/dsl/resources/ome/dsl/object.vm

```diff
@@ -1286,6 +1286,16 @@ implements java.io.Serializable, IObject
 
     public void set${prop.nameCapped}(${prop.fieldType} ${prop.name}) {
         $errorIfUnloaded
+## See table column type alterations in psql-footer.vm.
+#if(($type.shortname == "Laser" && ($prop.name == "wavelength")) || ($type.shortname == "LightSettings" && ($prop.name == "wavelength")) || ($type.shortname == "LogicalChannel" && ($prop.name == "emissionWave" || $prop.name == "excitationWave")) || ($type.shortname == "Pixels" && ($prop.name == "physicalSizeX" || $prop.name == "physicalSizeY" || $prop.name == "physicalSizeZ")) || ($type.shortname == "TransmittanceRange" && ($prop.name == "cutIn" || $prop.name == "cutOut")))
```

Comment:
Why did the field names need to be hard-coded? Do the mapping files not capture the types appropriately?

Reply:
They indeed do not capture range constraints on dimensioned values.

Reply:
Aha. Further dragging cleanup from the units migration. Half-tempted to say we should do this correctly but putting the necessary metadata into the mapping files.

Reply:
I certainly agree though we're unlikely to effect this anytime soon. Needs some design: for instance, perhaps best to break existing range constraints wholly out of `type` into a new attribute? Could create a card.

Reply:
Do you want to move the card indicated in the description to a new board to capture the follow-up work?

Reply:
This PR satisfies the current card. Created https://trello.com/c/s1XwJTmD/ accordingly.